src/issuance.rs

```diff
@@ -241,19 +243,22 @@ pub struct Prepared {
 /// Marker for an authorized bundle.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Signed {
-    signature: IssueAuthSig<ZSASchnorr>,
+    signature: BIP340IssueAuthSigWithInfo,
 }
 
 impl Signed {
     /// Returns the signature for this authorization.
-    pub fn signature(&self) -> &IssueAuthSig<ZSASchnorr> {
+    pub fn signature(&self) -> &BIP340IssueAuthSigWithInfo {
         &self.signature
     }
 
     /// Constructs a `Signed` from a byte array containing an `IssueAuthSig` in raw bytes.
     pub fn from_data(data: &[u8]) -> Self {
         Signed {
-            signature: IssueAuthSig::decode(data).unwrap(),
+            signature: BIP340IssueAuthSigWithInfo::new(
+                ORCHARD_ISSUE_INFO_V0,
+                IssueAuthSig::decode(data).unwrap(),
+            ),
```

Comment:
adjust function signature to include the Version.

Reply:
Done